webapp/src/DepCard.js

```diff
@@ -16,8 +16,9 @@ class DepCard extends PureComponent {
 
   blockerCount(nodes) {
     var count = 0;
-    for (var index in (this.props.dependencies || [])) {
-      if (true) {
+    var dependencies = this.props.dependencies || [];
+    for (var index in dependencies) {
+      if (Object.prototype.hasOwnProperty.call(dependencies, index)) {
```

Comment:
The suggested phrasing in the ESLint docs [is][1]:

    {}.hasOwnProperty.call(foo, key)

I don't see the point in instantiating a new object when we can use `Object.prototype` directly.  

And while we could probably use `dependencies.hasOwnProperty(index)` and similar without problems, both of the references I cite are guarding against a locally-overridden `hasOwnProperty`.  I'm unlikely to audit our dependencies for `hasOwnProperty` overrides ;), so erring on the side of verbose caution makes the most sense to me.


[1]: http://eslint.org/docs/rules/guard-for-in